### PR TITLE
N32: DIMS jobs

### PR DESCRIPTION
**Pull request by AngelOnFira**
_Part of #979_

### CODE REVIEW

**Comment by AngelOnFira** [comment]
@frnoreen does this look accurate? I'm not sure if the position is remote/hybrid/local

image

**Comment by FredrikNoren** [comment]
@AngelOnFira I'd say local/remote are the two options

**Comment by AngelOnFira** [comment]
> @AngelOnFira I'd say local/remote are the two options

I think `Stolkholm/remote` would be the best? `local` would be a bit ambiguous I think.

File: content/news/032/index.md [hunk 1]
```diff
@@ -426,6 +426,11 @@ useful information -->
 
 <!-- An optional section for new jobs related to Rust gamedev -->
 
+- [DIMS](https://www.dims.co/jobs)
+  (Stockholm/Hybrid Remote)
```
Comment (17cupsofcoffee) [inline content/news/032/index.md:430]:
```suggestion
  (Stockholm/Remote)
```

**Comment by AngelOnFira** [comment]
lgtm :+1:

**Review by 17cupsofcoffee — APPROVED** [review]

**Pull request merged**